File: cw/python/check_same_case.py
# https://www.codewars.com/kata/5dd462a573ee6d0014ce715b/train/python


def same_case(a, b):
    if a.isalpha() and b.isalpha():
        if (a.isupper() and b.isupper()) or (a.islower() and b.islower()):
            return 1
        else:
            return 0
    else:
        return -1


if __name__ == '__main__':
    a = input()
    b = input()
    print(same_case(a, b))

File: codewars/c/check_same_case.c
// https://www.codewars.com/kata/5dd462a573ee6d0014ce715b/train/c

#include <stdio.h>
#include <ctype.h>

int same_case (char a, char b) {
	if (isalpha(a) && isalpha(b)) {
		if ((isupper(a) && isupper(b)) || (islower(a) && islower(b))) return 1;
		else return 0;
	} else {
		return -1;
	}
}


int main(int argc, char *argv[]) {
	printf("%d\n", same_case('C', 'B'));
	printf("%d\n", same_case('b', 'a'));
	printf("%d\n", same_case('d', 'd'));
	printf("%d\n", same_case('A', 's'));
	printf("%d\n", same_case('c', 'B'));
	printf("%d\n", same_case('b', 'Z'));
	printf("%d\n", same_case('\t', 'Z'));
	printf("%d\n", same_case('H', ':'));
}
